8Knot/cache_manager/db_init.py

```diff
@@ -261,6 +261,72 @@ def _create_application_tables() -> None:
         )
         logging.warning("CREATED repo_files_query TABLE")
 
+        cur.execute(
+            """
+            CREATE UNLOGGED TABLE IF NOT EXISTS repo_languages_query(
+                id int,
+                programming_language text,
+                code_lines int,
+                files int
+            )
+            """
+        )
+        logging.warning("CREATED repo_languages_query TABLE")
+
+        cur.execute(
+            """
+            CREATE UNLOGGED TABLE IF NOT EXISTS package_version_query(
+                id int,
+                name text,
+                current_release_date text,
+                latest_release_date text,
+                libyear float4
+            )
+            """
+        )
+        logging.warning("CREATED package_version_query TABLE")
+
+        cur.execute(
+            """
+            CREATE UNLOGGED TABLE IF NOT EXISTS repo_releases_query(
+                id int,
+                release_name text,
+                release_created_at text,
+                release_published_at text,
+                release_updated_at text
+            )
+            """
+        )
+        logging.warning("CREATED repo_releases_query TABLE")
+
+        cur.execute(
+            """
+            CREATE UNLOGGED TABLE IF NOT EXISTS ossf_score_query(
+                id int,
+                name text,
+                score float4
+            )
+            """
+        )
+        logging.warning("CREATED ossf_score_query TABLE")
+
+        cur.execute(
+            """
+            CREATE UNLOGGED TABLE IF NOT EXISTS repo_info_query(
+                id int,
```

Comment:
is `id` == `repo_id` in this file? If so I think it should be named that way

Reply:
will that break things? I didnt know if id was still a keyword or if we want to change that when we change all of them

Reply:
Nah we can change it w/o a problem. In the query, I think it's `repo_id as id` so that'd just go back to `repo_id` and when you use the column in the analysis you'd just use `id` too.

Reply:
okay we'll go back and update that when we change all the names